FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -915,6 +973,20 @@ def setupTopoClusters(inputCells,
                                                  OutputLevel=INFO)
         TopAlg += [augmentClusterAlg]
 
+        if addPi0RecoTool:
+            from Configurables import PairCaloClustersPi0
+            Pi0RecoAlg = PairCaloClustersPi0(
+                "resolvedPi0FromClusterPair",
+                inClusters=augmentClusterAlg.outClusters.Path,
+                unpairedClusters="Unpaired" + augmentClusterAlg.outClusters.Path,
+                reconstructedPi0="ResolvedPi0Particle",
+                massPeak=0.122201, # values determined from a dedicated study
```

Comment:
Would be nice to add a link to a talk, maybe will be covered in Wednesday's talk?